### PR TITLE
Add privacy section on single use claims.

### LINKED ISSUE #15: Origin-bound claims are more privacy-aware
Single-use, origin bound claims are generally safer than long-lived claims. We should mention this in the privacy considerations section.

### PR DESCRIPTION
**Pull request by msporny**
Add a section around preferring single use claims when possible. Closes #15.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2009,12 +2009,28 @@
...
 
-        <p class="issue">
-Single-use, origin bound claims are generally safer than long-lived
-claims.
+        <p>
+An ideal privacy respecting system would only require information to be
+disclosed by the holder that is necessary for the interaction with the verifier.
+The verifier would then record that the disclosure requirement was met and
+forget any sensitive information that was disclosed. In many cases,
+things such as regulatory burden prevent this ideal system from being employed.
```
Comment (burnburn) [inline index.html]: things -> competing priorities ?

File: index.html [hunk 2]
```diff
@@ -2009,12 +2009,28 @@
...
+In other cases, long-lived identifiers prevent the desire for single use.
```
Comment (burnburn) [inline index.html]: suggest striking "the desire for"

**Comment by burnburn** [comment]
Also verify terminology:  claim vs. credential

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html]: Fixed.

**Comment by msporny** [comment]
Changed claim terminology to credential.

**Pull request merged**